pkg/virt-api/webhooks/mutating-webhook/mutators/vmi-mutator.go

```diff
@@ -85,6 +86,22 @@ func (mutator *VMIsMutator) Mutate(ar *admissionv1.AdmissionReview) *admissionv1
 			return webhookutils.ToAdmissionResponseError(err)
 		}
 
+		// To maintain backward compatibility, a mutation webhook was added.
```

Comment:
I don't think it is a good idea to silently truncate values, especially if it is a potentially unique identifier. I'd prefer to reject creation/update of a VMI if this is a hard error.